src/index.html

```diff
@@ -7,6 +7,39 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <h1>Calendar of January</h1>
+    <div class="calendar calendarDay--startDay--fri calendarDay--monthLength--31">
+      <div class="calendarDay"></div>
+      <div class="calendarDay"></div>
+      <div class="calendarDay"></div>
```

Comment:
it's better to name them as an element of calendar
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```